core/src/main/res/layout/activity_dev_menu.xml

```diff
@@ -25,46 +25,101 @@
         android:layout_height="wrap_content"
         android:background="@android:color/background_light"
         android:orientation="vertical"
-        android:layout_alignParentBottom="true"
-        android:padding="15dp">
+        android:layout_alignParentBottom="true">
 
-        <TextView
-            android:id="@+id/title_text_view"
+        <RelativeLayout
+            android:id="@+id/action_bar"
+            android:layout_width="match_parent"
+            android:layout_height="?android:attr/actionBarSize"
+            android:theme="@android:style/Widget.DeviceDefault.ActionBar.Solid"
+            android:layout_marginBottom="15dp">
+
+            <ImageButton
+                android:id="@+id/back_button"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:background="@drawable/ic_baseline_arrow_back_24"
+                android:layout_marginStart="20dp"
+                android:layout_marginLeft="20dp"
```

Comment:
I think you can remove `android:layout_marginLeft` since you've defined `android:layout_marginStart`.  `start` is better, since it allows support for RTL languages.   

Actually, I'd suggest replacing `Left` with `Start` and `Right` with `End` everywhere in this file.

Reply:
If I remove android:layout_marginLeft, it says "To support older versions than API 17 (project specifies 16) you should also add android:layout_marginLeft="20dp"". Should I be using a higher minimum API version (16 is the default)?

Reply:
Yeah no we want to maintain a minimum version of 16

Reply:
Ahh, yeah, we don't want to raise the minimum above 16.   Disregard!